data-prepper-core/src/main/java/com/amazon/dataprepper/parser/model/DataPrepperConfiguration.java

```diff
@@ -71,6 +78,15 @@ public String getPrivateKeyPassword() {
         return privateKeyPassword;
     }
 
+    public List<MeterRegistryType> getMetricsRegistries() {
+        return metricsRegistries;
```

Comment:
Naming - `metricsRegistries` to `meterRegistries`? Applies to variable definitions at top of class too.